templates/generic/person.js

```diff
@@ -3,27 +3,47 @@ import { html } from '/src/main.js';
 const TITLES = ['mr', 'ms', 'mrs', 'miss'];
 
 export default (prefix = 'person') => html`
-<div name="${prefix}" class="filed-set">
-    <div class="field">
-        <label class="field__name">Title</label>
-        <select name="${prefix}[title]">
-            ${ TITLES.map(t => html`<option value="${t}"> ${ t.toUpperCase() }</option>`) }
-        </select>
-    </div>
+    <div name="${prefix}" class="filed-set">
+        <div class="field">
+            <label class="field__name">Title</label>
+            <select
+                name="${prefix}[title]"
+                required>
+                ${ TITLES.map(t => html`<option value="${t}"> ${ t.toUpperCase() }</option>`) }
+            </select>
+        </div>
 
-    <div class="field">
-        <label class="field__name" for="${prefix}[first-name]">First Name</label>
-        <input type="text" name="${prefix}[first-name]" placeholder="Jane" required />
-    </div>
+        <div class="field">
+            <label
+                class="field__name"
+                for="${prefix}[first-name]">First Name</label>
+            <input
+                type="text"
+                name="${prefix}[first-name]"
+                pattern="^[A-Za-z]+$"
```

Comment:
Risky assertion, but fine for a demo.

Reply:
you're right, it shouldn't be any stricter than necessary. I'd rather remove pattern and message